integration/tests/graphql-migrations-postgres.ts

```diff
@@ -0,0 +1,47 @@
+import * as Knex from "knex";
+
+import { buildSchema } from 'graphql';
+import { migrateDB } from "../../packages/graphql-migrations/src";
+import { Operation } from "../../packages/graphql-migrations/src/diff/Operation";
+
+
+const dbConfig: Knex.Config = {
+  client: "pg",
+  connection: {
+    user: "postgresql",
+    password: "postgres",
+    database: "users",
+    host: "localhost",
+    port: parseInt(process.env.POSTGRES_PORT, 10) || 5432
+  }
+}
+
+const db = Knex(dbConfig);
+
+beforeEach(async () => {
+  await db.raw(`DROP SCHEMA public CASCADE;
+CREATE SCHEMA public;
+GRANT ALL ON SCHEMA public TO postgresql;
+GRANT ALL ON SCHEMA public TO public;`);
+})
+
+afterAll(() => db.destroy());
+
+test('consecutive migrations', async () => {
+  const model = buildSchema(`
+  """@model"""
+  type User {
+    """@id"""
+    _id: ID
+    name: String
+  }
+`)
```

Comment:
<3 Thanks!

Reply:
Not sure why you are still getting the error - your old database could have data in it that is preventing the column from being dropped.